releaseChecklist.md

```diff
@@ -0,0 +1,26 @@
+Release checklist
+- [ ] Check outstanding issues on JIRA and Github
+- [ ] Publish documentation (`updateDocs.sh`) from `develop` branch
+  - [ ] Copy docs folder to `gh-pages` branch
+  - [ ] Overwrite existing develop folder with docs folder on `gh-pages`
+  - [ ] Push changes to `gh-pages branch`
```

Comment:
This happens automatically right? After `bash updateDocs.sh` should we list all those boxes?

Reply:
I figured it might be good to add in case you might have to do it manually for one reason or another.

Reply:
That's a good point.